src/index.js

```diff
@@ -163,6 +166,47 @@ class Offline {
     this.serverlessLog('https://github.com/dherault/serverless-offline/issues');
   }
 
+  static get AWS() {
```

Comment:
- what purpose serves `static ApiGatewayManagementApi : class ...`? it only seems to be used in the manual test case?

Reply:
To expose `ApiGatewayManagementApi({ apiVersion, endpoint })` class to be able then to call `postToConnection({ConnectionId, Data})` in order to send messages from server to ws clients. Can't use the the regular aws-sdk (or at least I haven't figured out a way to do it) to make it POST to http://localhost:3001/@connections/{connectionId}.

Reply:
ah, gotcha. if I remember correctly, `ApiGatewayManagementApi` from the `aws-sdk` seems to work just fine. it would be a bummer if not. otherwise we'd need to branch out for local and server imports. which you did, in your manual test case. makes more sense now why you did it.

I'll double check to make sure.

Reply:
alrighty, you're right, `aws-sdk/ApiGatewayManagementApi` didn't work with the current setup. I noticed after some debugging that the api sends the request as `binary/octet-stream`, which hapi doesn't allow with it's default mime-type list.

you have to add: `config:{ payload: {  parse: false }}` to the `'/@connections/{connectionId}'` route, so the payload is not being parsed, then it works.

Reply:
The first obstacle to using aws-sdk is that it sends by default requests over SSL. Any recommendation on how to overcome this obstacle?

Reply:
I think the ApiGatewayManagementApi constructor argument has an option for `sslEnabled`. https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/ApiGatewayManagementApi.html doesn't say what the default is. what's the problem you are facing?

Reply:
OK, got it to work via vanilla AWS SDK. I won't use `sslEnabled` since it's error prone (someone can leave this off even in production). By using the URL `http://localhost:3001`, the SDK sends over HTTP and not HTTPS. 
I'll inject this URL value in the lambda function `event` object.

Reply:
Done. Current usage option:

`let endpoint=event.apiGatewayUrl;`

`if (!endpoint) endpoint = event.requestContext.domainName+'/'+event.requestContext.stage;`

`const apiVersion='2018-11-29';`

`return new AWS.ApiGatewayManagementApi({ apiVersion, endpoint });`